atc/builds/tracker_runner.go

```diff
@@ -24,14 +28,17 @@ type Notifications interface {
 }
 
 type TrackerRunner struct {
-	Tracker       BuildTracker
-	Notifications Notifications
-	Interval      time.Duration
-	Clock         clock.Clock
-	Logger        lager.Logger
+	Tracker          BuildTracker
+	Notifications    Notifications
+	Interval         time.Duration
+	Clock            clock.Clock
+	Logger           lager.Logger
+	LockFactory      lock.LockFactory
+	ComponentFactory db.ComponentFactory
 }
 
 func (runner TrackerRunner) Run(signals <-chan os.Signal, ready chan<- struct{}) error {
+	close(ready)
```

Comment:
why did we move the close to the top of the function? We think closing the ready chan signals to ifrit the runner is ready, but it looks like we're still doing some initializing after the call.

Reply:
just wanted to make it consistent with all the other runners